restful_api/tests/factories.py

```diff
@@ -33,25 +32,19 @@ class EmployeeFactory(factory.Factory):
     class Meta:
         model = Employee
 
-class OfficialEnrollFactory(factory.Factory):
+class EnrollFactory(factory.Factory):
     eng_id = factory.Sequence(lambda n: 1)
     course_id = factory.Sequence(lambda n: n)
-    # start_date = factory.Sequence(lambda n: "%d" % n)
-    # end_date = factory.Sequence(lambda n: "%d" % n)
+    trainer_id = factory.Sequence(lambda n: n)
     has_passed = factory.Sequence(lambda n: False)
+    is_official = factory.Sequence(lambda n: False)
     class Meta:
-        model = OfficialEnroll
+        model = Enroll
```

Comment:
I think can try linking to other factories depending on your SQLAlchemy schema? For example, this is how to handle course. For engineer and trainer might need more thinking

```
class EnrollFactory(factory.Factory):
    eng_id = factory.Sequence(lambda n: 1)
    course_id = factory.SelfAttribute('course.course_id')
    trainer_id = factory.Sequence(lambda n: n)
    has_passed = factory.Sequence(lambda n: False)
    is_official = factory.Sequence(lambda n: False)
    
    course = factory.SubFactory(CourseFactory)
    class Meta:
        model = Enroll
```

Then in `test_course.py` it will be -
```
enroll_one = enroll_factory(course=courses[0], has_passed=True, eng_id=employee.id) # Same thing with engineer
enroll_two = enroll_factory(course=courses[1], has_passed=False, eng_id=employee.id)

```